rest_framework/fields.py

```diff
@@ -59,7 +59,11 @@ def is_simple_callable(obj):
     if not (function or method):
         return False
 
-    args, _, _, defaults = inspect.getargspec(obj)
+    if six.PY2:
+        args, _, _, defaults = inspect.getargspec(obj)
+    else:
+        args, _, _, defaults = inspect.getfullargspec(obj)[:4]
+
```

Comment:
What would the work overhead to use `Signature` instead of `getfullargspec` ?
I'm asking because `getfullargspec` is deprecated from 3.5 so since we're working on that part, might be good to jump directly to the `Signature`.